### DISCUSSION TITLE
Help for display brightness settings

### DISCUSSION THREAD

**Discussion by pbranly** [Blackymas/NSPanel_HA_Blueprint#2025] [Q&A]
Hi
I am lost with the different settings 
Display brightness, dimdown and sleep
They don't do what I wish.
I would like to have a given  brightness when I touch the screen and a reduced brightness for the a given screen 
I could adjust the nspanel to have the hour only but this is not what I want.
Please help
Phil

**Comment by edwardtfn** [comment]
You have 3 dim settings:
1. Display brightness: When you touch the screen, it will go to that brightness level. If the panel was sleeping, it will also go to the wake-up page.
2. Dim: There is a time set in "Timeout dim". After that time without touching the screen, it will adjust the brightness to the value set in "Brightness Dim".
3. Sleep: When the panel is sleeping (after "Timeout Sleep" time), it will use this brightness. Most people will use 0% here, however, if you enable the time display on sleep, you need some brightness.

**Reply by pbranly** [reply]
Thanks it is working 
Phil
